docs/using-gitbase/examples.md

```diff
@@ -6,105 +6,97 @@
 SELECT refs.repository_id
 FROM refs
 NATURAL JOIN commits
-WHERE commits.commit_author_name = 'Javi Fontan' AND refs.ref_name = 'HEAD';
+WHERE commits.commit_author_name = 'Johnny Bravo'
+  AND refs.ref_name = 'HEAD';
 ```
 
 ## Get all the HEAD references from all the repositories
 
 ```sql
-SELECT * FROM refs WHERE ref_name = 'HEAD';
+SELECT *
+FROM refs
+WHERE ref_name = 'HEAD';
 ```
 
 ## First commit on HEAD history for all repositories
 
 ```sql
-SELECT
-    file_path,
-    ref_commits.repository_id
-FROM
-    commit_files
-NATURAL JOIN
-    ref_commits
-WHERE
-    ref_commits.ref_name = 'HEAD'
-    AND ref_commits.history_index = 0;
+SELECT file_path,
+       ref_commits.repository_id
+FROM commit_files
+NATURAL JOIN ref_commits
+WHERE ref_commits.ref_name = 'HEAD'
+  AND ref_commits.history_index = 0;
 ```
 
 ## Commits that appear in more than one reference
 
 ```sql
-SELECT * FROM (
-    SELECT COUNT(c.commit_hash) AS num, c.commit_hash
-    FROM ref_commits r
-    INNER JOIN commits c
-        ON r.repository_id = c.repository_id AND r.commit_hash = c.commit_hash
-    GROUP BY c.commit_hash
-) t WHERE num > 1;
+SELECT *
+FROM
+  (SELECT COUNT(c.commit_hash) AS num,
+          c.commit_hash
+   FROM ref_commits r
+   NATURAL JOIN commits c
+   GROUP BY c.commit_hash) t
+WHERE num > 1;
 ```
 
 ##  Get the number of blobs per HEAD commit
 
 ```sql
-SELECT COUNT(c.commit_hash), c.commit_hash
-FROM ref_commits as r
-INNER JOIN commits c
-    ON r.ref_name = 'HEAD'
-    AND r.repository_id = c.repository_id
-    AND r.commit_hash = c.commit_hash
-INNER JOIN commit_blobs cb
-    ON cb.repository_id = c.repository_id AND cb.commit_hash = c.commit_hash
-GROUP BY c.commit_hash;
+SELECT COUNT(commit_hash),
+       commit_hash
+FROM ref_commits
+NATURAL JOIN commits
+NATURAL JOIN commit_blobs
+WHERE ref_name = 'HEAD'
+GROUP BY commit_hash;
 ```
 
-## Get commits per committer, per month in 2015
+## Get commits per committer, per month in 2018
 
 ```sql
-SELECT COUNT(*) as num_commits, month, repo_id, committer_email
-FROM (
-    SELECT
-        MONTH(committer_when) as month,
-        r.repository_id as repo_id,
-        committer_email
-    FROM ref_commits r
-    INNER JOIN commits c
-        ON YEAR(c.committer_when) = 2015
-        AND r.repository_id = c.repository_id
-        AND r.commit_hash = c.commit_hash
-    WHERE r.ref_name = 'HEAD'
-) as t
-GROUP BY committer_email, month, repo_id;
+SELECT COUNT(*) AS num_commits,
+       MONTH,
+       repo_id,
+       committer_email
+FROM
+  (SELECT MONTH(committer_when) AS MONTH,
+          repository_id AS repo_id,
+          committer_email
+   FROM ref_commits
+   NATURAL JOIN commits
+   WHERE ref_name = 'HEAD'
+     AND YEAR(committer_when) = 2018 ) AS t
```

Comment:
could group by year to make this query more general?

Reply:
@ajnavarro: I've updated - take a look